yoda/run.go

```diff
@@ -29,6 +28,10 @@ const (
 	EventChannelCapacity = 2000
 )
 
+var (
+	protoCodec = band.MakeEncodingConfig().Marshaler
```

Comment:
Hmm....

Reply:
Legacy vs Proto

Reply:
I have changed `cdc` to use `Marshaler` instead of `Amino`. I tried to send oracle request and it seemed to work fine. However, I also have to change argument type from `sdk.Msg` to `codec.ProtoMarshaler` to be able to use with `MarshalBinaryBare()`.